dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java

```diff
@@ -338,25 +338,29 @@ private View createView(LayoutInflater inflater) {
         holder.setBackgroundColor(backgroundColorResourceId);
         View view = holder.getView(inflater, rootView);
 
+        if (holder instanceof ViewHolder) {
+            loopViewsRecursively(view);
+        }
+
         loopViewsRecursively(headerView);
```

Comment:
what do you think if we rename this method? like initClickListenerRecursively(view) to make it more clear?